spring-cloud-gcp-samples/spring-cloud-gcp-logging-sample/README.adoc

```diff
@@ -0,0 +1,37 @@
+= Spring Cloud GCP Logging Starter Example
+
+This sample application demonstrates using the
+link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-logging[Spring Cloud GCP Logging Starter] in your code.
+
+== Setup & Configuration
+1. Create a Google Cloud Platform Project
+1. Have the https://cloud.google.com/sdk/[Google Cloud SDK] installed and
+initialized and logged in with
+https://developers.google.com/identity/protocols/application-default-credentials[application
+default credentials].
+
+1. Check that the Stackdriver Logging API is enabled.
+
+== Run the Example
+Deploy the sample application from Maven:
+
+----
+mvn appengine:deploy
+----
+
+Browse to `http://[Your-Project-ID].appspot.com/log`. This will trigger the `ExampleController.log()` method,
+which uses a typical logger supporting Logback to write a line of text to the Stackdriver log.
+
+To see the logged lines, navigate to the Logging section of the Cloud Console.
+Select your App Engine deployment as the resource and look under the `spring.log` channel to see the
+logged text. You can further narrow down the log by request by using the following filtering text
```

Comment:
Maybe also add an instruction to "Expand the log entry labels and verify that `appengine.googleapis.com/trace_id` label is set to the same value for both log messages."

Reply:
done